packages/plugin-coinbase/src/plugins/tokenContract.ts

```diff
@@ -574,7 +574,7 @@ export const tokenContractPlugin: Plugin = {
         "Enables deployment, invocation, and reading of ERC20, ERC721, and ERC1155 token contracts using the Coinbase SDK",
     actions: [
         deployTokenContractAction,
-        invokeContractAction,
+      //  invokeContractAction,
```

Comment:
why is it now commented out?

Reply:
it is a security risk someone could call transfer on a smart contract and drain our funds

Reply:
We will reenable once we allow users to configure their own credentials

Reply:
let's leave TODO then